src/Task/ListItem.ts

```diff
@@ -16,4 +16,22 @@ export class ListItem {
             parent.children.push(this);
         }
     }
+
+    /**
+     * Parses the parents recursively until a {@link ListItem} with a `null` parent is found.
+     */
+    get root(): ListItem {
+        if (this.parent === null) {
+            return this;
+        }
+
+        return this.parent.root;
+    }
+
+    /**
+     * Checks whether this item has no parent.
+     */
```

Comment:
Again, this is a little bit like re-stating the implementation.

It's also good to link to related methods.

I will do these things.